core/src/main/scala/org/apache/gearpump/Message.scala

```diff
@@ -25,8 +25,9 @@ package org.apache.gearpump
  * message's timestamp.
  * @param msg Accept any type except Null, Nothing and Unit
  */
-case class Message(msg: Any, timestamp: TimeStamp = Message.noTimeStamp)
+case class Message(msg: Any, timestamp: TimeStamp = Message.currentTimeStamp)
 
 object Message {
   val noTimeStamp: TimeStamp = 0L
+  val currentTimeStamp: TimeStamp = System.currentTimeMillis()
```

Comment:
And I don't think it's right. The `currentTimeStamp` is just a `val` so once it is initiated, the value won't change anymore.